pkgs/applications/video/obs-studio/default.nix

```diff
@@ -198,9 +214,18 @@ stdenv.mkDerivation (finalAttrs: {
       video content, efficiently
     '';
     homepage = "https://obsproject.com";
-    maintainers = with maintainers; [ eclairevoyant jb55 materus fpletz ];
+    maintainers = with maintainers; [
+      eclairevoyant
+      jb55
+      materus
+      fpletz
+    ];
     license = with licenses; [ gpl2Plus ] ++ optional withFdk fraunhofer-fdk;
-    platforms = [ "x86_64-linux" "i686-linux" "aarch64-linux" ];
+    platforms = [
```

Comment:
The architecture should be changed to only list `x86_64-linux` because the binary build of CEF provided by upstream is x86_64 only.

Reply:
Marking this a draft for now. I was searching for binaries for other architectures but wasn't successful. It would be best if our version of CEF would be supported but I'm not sure what's the problem. Using the binary provided by OBS does fix it though.